src/arch/armv8/armv8-r/mpu.c

```diff
@@ -157,7 +157,7 @@ static inline mem_attrs_t mpu_entry_attrs(struct mp_region* mpr)
 static mpid_t mpu_entry_allocate()
 {
     mpid_t reg_num = INVALID_MPID;
-    for (mpid_t i = 0; i < mpu_num_entries(); i++) {
+    for (mpid_t i = 0; i < (ssize_t)mpu_num_entries(); i++) {
```

Comment:
Maybe we could cast this to `mpid_t`?

Reply:
Fixed in aef4397dde5adfee49fdfa9159b05c1b1ccfd493